docs/rules.adoc

```diff
@@ -0,0 +1,46 @@
+ifdef::env-github,env-browser[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+:outfilesuffix: .adoc
+endif::[]
+
+=== Rules
+:icons: font
+
+If you want to use arquillian functionality with some other runner. You can use Arquillian with Junit Rule using `ArquillianTestClass` and `ArquillianTest` Rules.
+
+In order to use full functionality of Arquillian you have to use both the rules i.e. `ArquillianTestClass` and
+`ArquillianTest`.
+
+==== How to use it?
+
+[source, java, indent=0]
+----
+public class HelloWorldTest
+{
+   @ClassRule
+   public static ArquillianTestClass arquillianTestClass = new ArquillianTestClass();
+
+   @Rule
+   public ArquillianTest arquillianTest = new ArquillianTest();
+
+   @Test
+   public void shouldNotBeNull()
+   {
+      Assert.assertNotNull(arquillianTest);
+   }
+}
```

Comment:
Let's use some more realistic test and new code formatting too. This test just shows then `new ArquillianTest()` works ;)